### DISCUSSION TITLE
Download link is not working, it just keeps checking and doesn't download.

### DISCUSSION THREAD

**Discussion by Mkcein** [bleachbit/bleachbit#1426] [General]
I'm trying to download both the installer version and the portable version from the official website but it's not completing the download. Could you please fix it?

**Comment by az0** [comment]
Could you post a screenshot? When you are typing a comment and GitHub, you
can just press control plus v to paste an image from clipboard.

Are you using bleachbit.org?

Can you download the installer or Linux packages from this site or is it
just a portable?

**Reply by Mkcein** [reply]
I made a video demonstrating the problem, the problem only occurs with files for Windows, for the linux version I can download normally.
Yes, I used the official website to download. I live in Brazil.

https://youtu.be/aIofoo_-g58

**Comment by az0** [comment]
What is the URL when the screen states that "the request is being verified"?

**Reply by Mkcein** [reply]
Installer:  https://www.bleachbit.org/download/file/t?file=BleachBit-4.4.2-setup.exe

Portable: https://www.bleachbit.org/download/file/t?file=BleachBit-4.4.2-portable.zip

Both don't work

**Comment by az0** [comment]
Do you Tor, a VPN, ad/script blocker extensions, or anything like that?

Which browser are you using?

**Reply by Mkcein** [reply]
I don't use VPN or blocking script, I turned off Kaspersky antivirus.

I tried it in OPERA and Microsoft EDGE browsers, the strange thing is that I can normally download any .zip from any site and any .exe from any site, I tried with your Linux files and they all worked normally.

**Reply by Mkcein** [reply]
I asked a friend of mine to download it and it worked at his house, he sent me the direct download link so I managed to download it, but I can't do it through this redirector, honestly I don't know what it could be but I managed to download it using this direct link :

https://download.bleachbit.org/BleachBit-4.4.2-setup.exe

**Comment by az0** [comment]
I changed a server setting. Try again.

If it does not work, please email me the time (if different than when you sent the email) and your IP address to ahz001 at gmail dot com, so I can check the server log.

**Comment by az0** [comment]
The message about the request being verified looks like a firewall on my end. I got your email, but there is nothing in the firewall activity about a challenge or block with that IP address in last 24 hours.

I also checked the firewall log by time, ignoring IP address. The closest things are more than 10 minutes. One is Tor, and the other is Russian federation, so that doesn't look like you either.

I wonder about you testing another device from the same network. This could be for example trying The download from your cell phone when connected the same Wi-Fi as the laptop you tested.

Another test would be if you could take your laptop to another network such as a coffee shop, school, or work.
